bot.py

```diff
@@ -37,6 +39,7 @@
 
 intents = discord.Intents.all()
 logger = logging.getLogger(__name__)
+SYNC_COMMAND_NAME = "sync"
```

Comment:
> I don't think we'll change the name of the sync command.

True, but we need the sync name to prevent showing the "please use slash command" message (looking up the references for `SYNC_COMMAND_NAME` shows the two locations where it is read, compared to having two magic strings in either location).